PowerEditor/src/stylers.model.xml

```diff
@@ -656,6 +656,8 @@
             <WordsStyle name="OPERATOR" styleID="10" fgColor="000080" bgColor="FFFFFF" fontName="" fontStyle="1" fontSize="" />
             <WordsStyle name="IDENTIFIER" styleID="11" fgColor="000000" bgColor="FFFFFF" fontName="" fontStyle="0" fontSize="" />
             <WordsStyle name="COMMENTBLOCK" styleID="12" fgColor="008000" bgColor="FFFFFF" fontName="" fontStyle="0" fontSize="" />
+            <WordsStyle name="STRING NOT CLOSED" styleID="13" fgColor="000000" bgColor="E0C0E0" fontName="" fontStyle="0" fontSize="" eolfilled="1" />
+            <WordsStyle name="USER DEFINED" styleID="14" fgColor="407090" bgColor="FFFFFF" fontName="" fontStyle="1" fontSize="" keywordClass="instre2" />

```

Comment:
As also/already mentioned in #2585:
1.  I guess separate PR's would have been better.
2.  Furthermore, why add a separate style for "USER DEFINED" keywords when it is already possible to enter user-defined keywords as part of the "KEYWORDS" style?